maxim/logger/pydantic_ai/client.py

```
@@ -0,0 +1,735 @@
+            "id": trace_id,
+            "name": name,
+            "tags": {"session": "true"},
+            "input": "Session started",
+        })
+        _global_maxim_trace.set(_session_trace)
+        scribe().debug(f"[MaximSDK] Started session trace: {trace_id}")
+    
+    return _session_trace
+
+
+def end_session_trace(maxim_logger: Logger):
```

Ruff (0.12.2)

Missing return type annotation for public function `end_session_trace`

Add return type annotation: `None`

(ANN201)